concepts/felts/about.md

```diff
@@ -1 +1,26 @@
 # Felt Type
+
+# Scalar Types in Cairo
```

Comment:
I would remove this "scalar types" section, as we'd like to concept to focus solely on felts. We can just leave link to the "Scalar Types" section of The Cairo Book somewhere in felt's description.